src/strategies/path-balance-staked-and-locked/index.ts

```diff
@@ -0,0 +1,63 @@
+import { BigNumberish } from '@ethersproject/bignumber';
+import { formatUnits } from '@ethersproject/units';
+import { Multicaller } from '../../utils';
+
+export const author = 'PathDAO';
+export const version = '0.1.0';
+
+const tokenAbi = [
+  'function balanceOf(address account) external view returns (uint256)'
+];
+
+const stakingAbi = [
+  'function balanceOf(address account) external view returns (uint256)'
+];
```

Comment:
These are duplicate code, can't this just be `const abi`?

Reply:
yeah this will be fine let me just change that